src/lms-ds-loader/tests_integration_sql/conftest.py

```diff
@@ -0,0 +1,78 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+from typing import Tuple
+import pytest
+from unittest.mock import MagicMock
+from pandas import DataFrame
+from sqlalchemy import create_engine
+from sqlalchemy.engine.base import Engine, Connection, Transaction
+
+from edfi_lms_ds_loader.migrator import migrate
+from edfi_lms_ds_loader.mssql_lms_operations import MssqlLmsOperations
+
+
+def _new_mssql_engine() -> Engine:
+    return create_engine(
+        "mssql+pyodbc://localhost,1433/test_integration_lms_toolkit?driver=ODBC+Driver+17+for+SQL+Server?Trusted_Connection=yes",
+    )
+
+
+@pytest.fixture(scope="session")
+def mssql_connection():
+    """
+    Fixture that sets up a connection to use, and migrate the tables.
+    Assumes existence of local SQLServer DB named 'test_integration_lms_toolkit'
+    """
+    engine = _new_mssql_engine()
+    migrate(engine)
+
+    connection = engine.connect()
+    yield connection
+    connection.close()
```

Comment:
Nice usage of `yield`. I've never really figured out when to use that before. I've wanted this exact pattern before and didn't think of it. Instead I tend to pass a callback into this function, which can be called at line 33. I like this `yield` pattern better, since it is easier to reason about program flow without callbacks.

@lalrg please also take a look at this.